tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs

```diff
@@ -565,6 +566,115 @@ impl ServerKey {
+    /// a signed ciphertext and a signed clear value and assigns the
+    /// result to the input ciphertext.
+    ///
+    /// # Note
+    ///
+    /// - This division rounds (truncates) the quotient towards 0
+    pub fn signed_scalar_div_assign_parallelized<T>(
+        &self,
+        numerator: &mut SignedRadixCiphertext,
+        divisor: T,
+    ) where
+        T: SignedReciprocable,
+        <<T as SignedReciprocable>::Unsigned as Reciprocable>::DoublePrecision: Send,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        *numerator = self.unchecked_signed_scalar_div_parallelized(numerator, divisor);
+    }
+
+    /// Computes and returns the quotient of the division between
+    /// a signed ciphertext and a signed clear value.
+    ///
+    /// # Note
+    ///
+    /// - This division rounds (truncates) the quotient towards 0    /// # Note
+    /// - If you need both the quotient and remainder use [Self::signed_scalar_div_rem_parallelized]
+    ///   instead.
+    pub fn signed_scalar_div_parallelized<T>(
+        &self,
+        numerator: &SignedRadixCiphertext,
+        divisor: T,
+    ) -> SignedRadixCiphertext
+    where
+        T: SignedReciprocable,
+        <<T as SignedReciprocable>::Unsigned as Reciprocable>::DoublePrecision: Send,
+    {
+        let mut result = numerator.clone();
+        self.signed_scalar_div_assign_parallelized(&mut result, divisor);
+        result
+    }
+
+    /// Computes and returns the remainder of the division between
+    /// a signed ciphertext and a signed clear value.
+    ///
+    /// # Note
+    ///
+    /// - If you need both the quotient and remainder use [Self::signed_scalar_div_rem_parallelized]
+    ///   instead.
+    pub fn signed_scalar_rem_parallelized<T>(
+        &self,
+        numerator: &SignedRadixCiphertext,
+        divisor: T,
+    ) -> SignedRadixCiphertext
+    where
+        T: SignedReciprocable + ScalarMultiplier,
+        <<T as SignedReciprocable>::Unsigned as Reciprocable>::DoublePrecision: Send,
+    {
+        let mut result = numerator.clone();
+        self.signed_scalar_rem_assign_parallelized(&mut result, divisor);
+        result
+    }
+
+    /// Computes the remainder of the division between
+    /// a signed ciphertext and a signed clear value and assigns the
+    /// result to the input ciphertext.
+    pub fn signed_scalar_rem_assign_parallelized<T>(
+        &self,
+        numerator: &mut SignedRadixCiphertext,
+        divisor: T,
+    ) where
+        T: SignedReciprocable + ScalarMultiplier,
+        <<T as SignedReciprocable>::Unsigned as Reciprocable>::DoublePrecision: Send,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        let (_quotient, remainder) =
+            self.unchecked_signed_scalar_div_rem_parallelized(numerator, divisor);
```

Comment:
no unchecked rem parallelized ? not a blocker

Reply:
added it as its a simple call to div_rem